20250401-kubecon-london/workshop/02-explore-workspaces/README.md

```diff
@@ -112,7 +112,7 @@ kubectl ws tree
 
 You should get output similar to this:
 
-```shell
+```shell-session
```

Comment:
`shell-session` is a kind of code block that recognizes a session of prompts, commands and outputs, and has offers highlighting different to the regular `bash/shell/console`.

```shell-session
root# echo hello
hello
```

GitHub doesn't seem to care, but mkdocs recognizes it.